drivers/gpu/drm/vc4/vc4_vec.c

```diff
@@ -45,6 +45,7 @@
 #define VEC_CONFIG0_YDEL(x)		((x) << 26)
 #define VEC_CONFIG0_CDEL_MASK		GENMASK(25, 24)
 #define VEC_CONFIG0_CDEL(x)		((x) << 24)
+#define VEC_CONFIG0_SECAM_STD		BIT(21)
```

Comment:
More magic fields. CONFIG0 bits 23:19 are defined a reserved, write as 0. Then again it also doesn't say how to select SECAM mode!

Reply:
Well, that's another product of me just messing around with this register. I read this header, saw that `VEC_FCW_SECAM_B` and `VEC_SECAM_GAIN_VAL` registers and was intrigued, especially since nothing else referred to SECAM and I don't think support for SECAM on the Pi was ever seen before. I started flipping bits in this register and when I set this one, again, poof, SECAM appeared.

I empirically confirmed that this mode is detected as SECAM and looks right on a couple of modern devices:
* Fushicai USBTV007 grabber
* Samsung UE40E6800S
* Panasonic TX-L42ET5E
* Samsung UE65MU8002 (through an RF modulator, this TV does not have any baseband analog inputs)
Again, @Sobakin76 also confirmed this working on their more period-accurate Sony PVM-14N5MDE CRT: https://github.com/raspberrypi/firmware/issues/811#issuecomment-802360785